scripts/killall_sglang.sh

```diff
@@ -8,7 +8,9 @@ kill -9 $(ps aux | grep 'sglang::' | grep -v 'grep' | awk '{print $2}') 2>/dev/n
 kill -9 $(ps aux | grep 'sglang.launch_server' | grep -v 'grep' | awk '{print $2}') 2>/dev/null
 kill -9 $(ps aux | grep 'sglang.bench' | grep -v 'grep' | awk '{print $2}') 2>/dev/null
 
-# Clean all GPU processes if any argument is provided
-if [ $# -gt 0 ]; then
-    kill -9 $(nvidia-smi | sed -n '/Processes:/,$p' | grep "   [0-9]" | awk '{print $5}') 2>/dev/null
-fi
+# Clean all GPU processes
+kill -9 $(nvidia-smi | sed -n '/Processes:/,$p' | grep "   [0-9]" | awk '{print $5}') 2>/dev/null
```

Comment:
I didn't enable it by default because, in some environments, we shouldn't terminate other processes.

Reply:
For example, in our development environment, we share an H100 node, and others might use the GPU to run other tasks.

Reply:
This is why I didn't set it as default, but it's also easy to enable when we want to, just by adding an extra parameter.

Reply:
Hmm OK, lemme update it then.

Reply:
@zhyncs put back the if condition and enabled killing for the router tests.